plugin.json

```diff
@@ -171,6 +171,12 @@
                       "value": "UserPrincipalName"
                   }
               ]         
+        },{
+            "key": "enableRhs",
+            "display_name": "Enable RHS",
+            "type": "bool",
+            "help_text": "When true, user will be able to view RHS (right hand sidebar).",
+            "default": false
```

Comment:
is this confirmed from Mike that initially don't show to anyone?

Reply:
yes